src/components/side_panel/components/radio_selection/radio_selection.ts

```diff
@@ -10,9 +10,10 @@ interface Choice {
 
 interface Props {
   choices: Choice[];
-  onChange: (value: string) => void;
+  onChange: (value: any) => void;
```

Comment:
`any` :eyes:

Reply:
yeah, in this particular case I want to accept any pivot value (string/boolean/integer/null), but we could very well have POJO or whatever, it doesn't go through the DOM